pkg/cmd/util/clientcmd/factory.go

```diff
@@ -201,7 +201,11 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 			if !ok {
 				return nil, errors.New("provided options object is not a BuildLogOptions")
 			}
-			buildsForBCSelector := labels.SelectorFromSet(map[string]string{buildapi.DeprecatedBuildConfigLabel: t.Name})
+			buildConfigLabels := labels.Set{
+				buildapi.BuildConfigLabel: t.Name,
```

Comment:
@liggit by server side I mean etcd server, not api server. Selectors don't
filter data in etcd is my point. So using a selector vs fetching all saves
you one hop but you're still paying the other one for all the data
regardless of how good your selector is.

Ben Parees | OpenShift
On Nov 5, 2015 1:53 PM, "Jordan Liggitt" <notifications@github.com> wrote:

> In pkg/cmd/util/clientcmd/factory.go
> <https://github.com/openshift/origin/pull/5733#discussion_r44052071>:
>
> > @@ -201,7 +201,11 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
> >  			if !ok {
> >  				return nil, errors.New("provided options object is not a BuildLogOptions")
> >  			}
> > -			buildsForBCSelector := labels.SelectorFromSet(map[string]string{buildapi.DeprecatedBuildConfigLabel: t.Name})
> > +			buildConfigLabels := labels.Set{
> > +				buildapi.BuildConfigLabel: t.Name,
>
> no, selectors are server-side
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5733/files#r44052071>.
>